src/chromo/kinematics.py

```diff
@@ -272,3 +302,56 @@ def __init__(self, energy, particle1, particle2):
                 f"{energy!r} is neither a number nor one of "
                 "TotalEnergy, KinEnergy, Momentum"
             )
+
+
+class BeamData:
```

Comment:
Make this a dataclass. You get the init then for free, ie. you don't have to write it yourself.

Reply:
Remove attributes mothers, daughters, status. These are all constant values, they should not be settable.

Reply:
Actually, going further, also vx, vy, vz, vt have predictable constant values. So these do not need to be part of the class either.

Reply:
Ok. It uses dict now.